### PR TITLE
IsPostcodeWithinRadii endpoint

### PR DESCRIPTION
**Pull request by william-thomas**

### CODE REVIEW

File: AddressService/AddressService.AzureFunction/Startup.cs [hunk 1]
```diff
@@ -53,6 +53,7 @@ public override void Configure(IFunctionsHostBuilder builder)
             }
             else
             {
+                configBuilder.AddUserSecrets(Assembly.GetExecutingAssembly(), false);
```
Comment (jawwadmukhtargmail) [inline AddressService/AddressService.AzureFunction/Startup.cs:56]: this looks wrong

File: AddressService/AddressService.AzureFunction/Startup.cs [hunk 2]
```diff
@@ -110,6 +114,7 @@ public override void Configure(IFunctionsHostBuilder builder)
 
             builder.Services.AddMediatR(typeof(GetPostcodeHandler).Assembly);
             builder.Services.AddMediatR(typeof(GetNearbyPostcodesHandler).Assembly);
```
Comment (jawwadmukhtargmail) [inline AddressService/AddressService.AzureFunction/Startup.cs:116]: Don't think you need to add one per handler. The .Assembly should take of all mediators within a project

File: AddressService/AddressService.Core/Dto/NearestPostcodeDto.cs [hunk 3]
```diff
@@ -1,13 +1,16 @@
-﻿using Newtonsoft.Json;
+﻿using System.Runtime.Serialization;
+using Newtonsoft.Json;
 
 namespace AddressService.Core.Dto
 {
     public class NearestPostcodeDto
     {
         [JsonProperty("a")]
+        [DataMember(Name = "a")]
```
Comment (jawwadmukhtargmail) [inline AddressService/AddressService.Core/Dto/NearestPostcodeDto.cs:9]: why the name a?

File: AddressService/AddressService.Core/Dto/NearestPostcodeDto.cs [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
         public string Postcode { get; set; }
 
         [JsonProperty("b")]
+        [DataMember(Name = "b")]
```
Comment (jawwadmukhtargmail) [inline AddressService/AddressService.Core/Dto/NearestPostcodeDto.cs:13]: why the name b?

File: AddressService/AddressService.Core/Utils/CompressionUtils.cs [hunk 5]
```diff
@@ -42,13 +56,26 @@ public static string Unzip(byte[] bytes)
             {
                 using (var gs = new GZipStream(msi, CompressionMode.Decompress))
                 {
-                    //gs.CopyTo(mso);
```
Comment (jawwadmukhtargmail) [inline AddressService/AddressService.Core/Utils/CompressionUtils.cs:45]: can we get rid of this comment

Comment (william-thomas) [reply AddressService/AddressService.Core/Utils/CompressionUtils.cs:45]: the comment is removed in the latest comment already

File: AddressService/AddressService.AzureFunction/Startup.cs (hunk 1 above)
Comment (william-thomas) [reply AddressService/AddressService.AzureFunction/Startup.cs:56]: you are right...

File: AddressService/AddressService.AzureFunction/Startup.cs (hunk 2 above)
Comment (william-thomas) [reply AddressService/AddressService.AzureFunction/Startup.cs:116]: true

File: AddressService/AddressService.Core/Dto/NearestPostcodeDto.cs (hunk 3 above)
Comment (william-thomas) [reply AddressService/AddressService.Core/Dto/NearestPostcodeDto.cs:9]: to make it compress and serialise faster (the property name will be shorter) - i've tested that it does make it faster.

File: AddressService/AddressService.Core/Dto/NearestPostcodeDto.cs (hunk 4 above)
Comment (william-thomas) [reply AddressService/AddressService.Core/Dto/NearestPostcodeDto.cs:13]: to make it faster

**Pull request merged**